src/Approvals/SFA.DAS.Approvals/Application/BulkUpload/Commands/BulkUploadAddDraftApprenticeshipsCommandHandler.cs

```diff
@@ -37,6 +37,7 @@ public async Task<GetBulkUploadAddDraftApprenticeshipsResult> Handle(BulkUploadA
             {
                 BulkUploadDraftApprenticeships = command.BulkUploadAddDraftApprenticeships,
                 ProviderId = command.ProviderId,
+                LogId = command.FileUploadLogId,
```

Comment:
as above but test class is 'WhenAddingBulkUploadDraftApprenticeships'

Reply:
Thanks, added that to the existing test